tools/apigee-x-trial-provision/README.md

```diff
@@ -127,3 +127,180 @@ the `CERTIFICATES` environment variable:
 |`generated`|Auto-Generated Self-Signed Certs|$RUNTIME_HOST_ALIAS or else $ORG-eval.apigee.net|
 |`supplied`|User-supplied in `RUNTIME_SSL_CERT` and `RUNTIME_SSL_KEY`|$RUNTIME_HOST_ALIAS or else $ORG-eval.apigee.net|
 <!-- markdownlint-enable MD013 -->
+
+## Apigee X instance with Custom Networking Provisioning
+<!-- markdownlint-disable MD013 -->
+
+## VPC Network Planning
+
+There is always a step to plan your network with regards to CIDR allocations.
+
+GCP/GKE
+CIDR: 10.0.0.0/14 CIDR IP Range: 10.0.0.0 - 10.3.255.255
+
+CIDR: 10.0.0.0/16 CIDR IP Range: 10.0.0.0 - 10.0.255.255
+
+NOTE: GCP recommended naming convention: <https://cloud.google.com/solutions/best-practices-vpc-design>
+
+### Bastion VM in a default network
+
+As Apigee X instance provisioning is a long-running operation, we would recommend to use your working PC terminal or to provision a bastion VM. Bastion VM is also useful for troubleshooting.
+
+1. In the GCP Console, select Compute Engine/VM instances hamburger menu item.
```

Comment:
can we link the gcp docs for how to create a vm?

Reply:
discussed out-of-the-band.